adapters/openx/openx.go

```diff
@@ -32,6 +33,12 @@ type openxRespExt struct {
 	FledgeAuctionConfigs map[string]json.RawMessage `json:"fledge_auction_configs,omitempty"`
 }
 
+type oxBidExt struct {
+	DspId   *string `json:"dsp_id,omitempty"`
+	BrandId *string `json:"brand_id,omitempty"`
+	BuyerId *string `json:"buyer_id,omitempty"`
```

Comment:
minor

not critical, but I don't think we need pointers here, more common to define just strings and check for empty string

```suggestion
	DspId   string `json:"dsp_id,omitempty"`
	BrandId string `json:"brand_id,omitempty"`
	BuyerId string `json:"buyer_id,omitempty"`
```

Reply:
changed, thanks for suggestion